ui/components/src/TransformsEditor/TransformEditor.tsx

```diff
@@ -0,0 +1,271 @@
+        multiple
+        id="merge-columns-columns"
+        sx={{ width: '100%' }}
+        options={[]}
+        value={value.spec.plugin.spec.columns}
+        renderInput={(params) => <TextField {...params} variant="outlined" label="Columns" required />}
+        onChange={(_, columns) => {
+          onChange({
+            ...value,
+            spec: {
+              ...value.spec,
+              plugin: { ...value.spec.plugin, spec: { ...value.spec.plugin.spec, columns: columns } },
+            },
+          });
+        }}
+      />
+
+      <TextField
+        id="merge-columns-name"
+        variant="outlined"
+        label="Output Name"
+        value={value.spec.plugin.spec.name}
+        sx={{ width: '100%' }}
+        onChange={(e) => {
+          onChange({
+            ...value,
+            spec: {
+              ...value.spec,
+              plugin: { ...value.spec.plugin, spec: { ...value.spec.plugin.spec, name: e.target.value } },
+            },
+          });
+        }}
+        required
+      />
+      <FormControlLabel
+        label="Enabled"
+        labelPlacement="start"
+        control={
+          <Switch
+            value={!value.spec.disabled ?? true}
+            checked={!value.spec.disabled ?? true}
+            onChange={(e) =>
+              onChange({
+                ...value,
+                spec: { ...value.spec, disabled: !e.target.checked },
+              })
+            }
+          />
+        }
+      />
+    </Stack>
+  );
+}
+
+function MergeIndexedColumnsTransformEditor({
+  value,
+  onChange,
+}: TransformSpecEditorProps<MergeIndexedColumnsTransformSpec>) {
+  return (
+    <Stack direction="row">
+      <TextField
+        id="merge-indexed-columns"
+        variant="outlined"
+        label="Column"
+        placeholder="Example: 'value' for merging 'value #1', 'value #2' and 'value #...'"
+        value={value.spec.plugin.spec.column}
+        sx={{ width: '100%' }}
+        onChange={(e) => {
+          onChange({
+            ...value,
+            spec: { ...value.spec, plugin: { ...value.spec.plugin, spec: { column: e.target.value } } },
+          });
+        }}
+        required
+      />
+      <FormControlLabel
+        label="Enabled"
+        labelPlacement="start"
+        control={
+          <Switch
+            value={!value.spec.disabled ?? true}
+            checked={!value.spec.disabled ?? true}
+            onChange={(e) =>
+              onChange({
+                ...value,
+                spec: { ...value.spec, disabled: !e.target.checked },
+              })
+            }
+          />
+        }
+      />
+    </Stack>
+  );
+}
+
+function MergeSeriesTransformEditor({ value, onChange }: TransformSpecEditorProps<MergeSeriesTransformSpec>) {
+  return (
+    <Stack direction="row">
+      <FormControlLabel
+        label="Enabled"
+        labelPlacement="start"
+        control={
+          <Switch
+            value={!value.spec.disabled ?? true}
```

Comment:
```suggestion
            value={!value.spec.disabled}
```
`!undefined` returns `true`, so `??` is never evaluated